src/modules/articles.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const Articles = {
+  async index() {
+    let response;
+    try {
+      const result = await axios.get(`/articles`);
+      response = result.data.articles;
+    } catch (error) {
+      response = error.response.data.error_message;
+    } finally {
+      return response;
```

Comment:
right now you are setting response in the state of articles, which should be an array that you pass to the flatlist. Have you tried what happens in you get an error? You can do some manual testing to trigger a 500 status for example